scripts/zephyr_module.py

```diff
@@ -39,6 +39,27 @@
       kconfig:
         required: false
         type: str
+  tests:
+    required: false
+    type: map
+    mapping:
+      path:
+        type: str
+        required: false
+  samples:
+    required: false
+    type: map
+    mapping:
+      path:
+        type: str
+        required: false
+  boards:
```

Comment:
Why did you add boards here, since that should really be part of the `sample.yaml` or `test.yaml`?